src/main/webapp/assets/js/feedScript.js

```diff
@@ -959,20 +1055,27 @@ async function checkLocationAndSubmit() {
         `We couldn't find address '${modalLocation}'. ` +
         'Please check your address for errors. ' +
         'If you wish to submit anyway, no pin will be added to the map.';
-      modalForm.insertBefore(modalError, document.getElementById('modal-submit'));
+      modalButtonDiv.insertBefore(modalError, document.getElementById('modal-submit'));
       submitModalButton.disabled = false;
     // Else, if the invalid address is the same as we last checked
     // or the address is just plain valid, then add the post to the Datastore.
     } else {
       const modalError = document.getElementById('modal-map-error');
       if (modalError) {
-        modalForm.removeChild(modalError);
+        modalButtonDiv.removeChild(modalError);
       }
 
       // (0,0) denotes a nonexistent lat/long, since it's a location in the ocean + is falsy.
       const lat = latLngResult ? latLngResult.lat : 0;
       const lng = latLngResult ? latLngResult.long : 0;
-      url = `/postData?collegeId=${collegeId}&lat=${lat}&lng=${lng}`;
+
+      const baseUrl = await getBlobstoreUrl();
+      if (!baseUrl) {
+        alert('Server Error. Try again later.');
+        submitModalButton.disabled = false;
+        return;
+      }
+      url = baseUrl + '?collegeId=' + collegeId + '&lat=' + lat + '&lng=' + lng;
```

Comment:
baseUrl + `?collegeId=${collegeId}&lat=${lat}&lng=${lng}`;`

Would that work?

Reply:
I was getting an error earlier, so I changed it. Good catch!